lsp-methods.el

```diff
@@ -45,7 +45,8 @@
   (response-handlers (make-hash-table :test 'eq) :read-only t)
 
   (string-renderers '())
-  (last-id 0))
+  (last-id 0)
+  (restart-command nil :read-only t))
```

Comment:
Please add a comment what `restart-command` is (i.e. its type and expected behavior).